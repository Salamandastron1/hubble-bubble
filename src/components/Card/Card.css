.card {
  display: grid;
  justify-items: center;
  background-size: cover;
  background: no-repeat center;
  height: 400px;
  width: 400px;
  margin: 20px;
  box-shadow: 0 20px 30px rgba(0, 0, 0, 0.2),
  inset 0px 10px 30px 5px #ebc;
  border-radius: 50%;
  align-items: center;
  transition: all .2s ease-in-out;
}
.card-name {
  text-decoration: none;
  color: white;
  font-family: 'Titan One';
  margin: 0;
  background: #000000b5;
  padding: 10px;
  text-align: center;
}
.link {
  text-decoration: none;
}

.card:hover {
  transform: scale(1.1);
}
