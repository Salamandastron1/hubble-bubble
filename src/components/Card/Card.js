import React from 'react'
import { Link } from 'react-router-dom'


const Card = ({star}) => {
  const { name, image_files, description, id } = star
  return (
    <Link to={`/studytime/${id}`}>
      <article className='card'>
        <h4>{name}</h4>
        <img src={image_files} height='400' width='500'/>
      </article>
    </Link>
  )
}

export default Card;