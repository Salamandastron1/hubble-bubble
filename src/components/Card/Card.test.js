import Card from './Card'
import React from 'react'
import { shallow } from 'enzyme'

describe('Card', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<Card star={{name: 'name', image_files: 'meow', id: 4, description: 'derp'}}/>)

    expect(wrapper).toMatchSnapshot()
  })
})