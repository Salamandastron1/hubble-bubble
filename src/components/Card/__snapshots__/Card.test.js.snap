// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should match the snapshot 1`] = `
<Link
  className="link"
  replace={false}
  to="/hubble-bubble/studytime/4"
>
  <article
    className="card"
    style={
      Object {
        "backgroundImage": "url(meow",
      }
    }
  >
    <h4
      className="card-name"
    >
      name
    </h4>
  </article>
</Link>
`;
